src/io.c

```diff
@@ -765,6 +765,29 @@ UInt CloseOutputLog ( void )
     return 1;
 }
 
+
+// Locate the previously opened stdout/stderr; return NULL if none found
+static TypOutputFile * findStdout(void)
+{
+    TypOutputFile * output = IO()->Output;
+    while (output && output->prev) {
+        if (!output->isstream && (output->file == 1 || output->file == 3))
```

Comment:
So right now we treat `*stdout*` and `*errout*` as the same device in almost all ways; e.g. they share the same formatting status and line buffer. The only difference at this point really is whether we print to the system's `stdout` or `stderr`.

But perhaps we shouldn't do it like this; perhaps we should only sync any open instances with `*stdout*` with each other, and separately `*errout*`; the only thing to sync between *both* would be the cursor position.

But even there, we may need to be careful: what if `stdout` and `stderr` actually go to different devices? E.g. stdout might be connected to one terminal, but stderr to another (I just verified that this works fine).

Soooo... any syncing between `*stdout*` and `*errout*` state (whether it is all state, or just the cursor position) likely should also depend on whether the system's `stdout` and `stderr` point to the same device; that's a check we can already add to `InitSysFiles` I guess (all the relevant info is already available there).